src/store/modules/product/mutations.ts

```diff
@@ -4,7 +4,9 @@ import * as types from './mutation-types'
 
 const mutations: MutationTree <ProductState> = {
   [types.PRODUCT_CURRENT] (state, payload) {
-    state.current = payload.product
+    payload.products.map((curr: any) => {
```

Comment:
I think we should provide a more auto explanatory name `curr` to `product`.

Reply:
Yes Sir,
Change the name from curr to product.
Thank you sir for reviewing my code.